listplayer.go

```diff
@@ -136,6 +143,28 @@ func (lp *ListPlayer) MediaState() (MediaState, error) {
 	return MediaState(state), getError()
 }
 
+// Previous plays previous item from media list.
+func (lp *ListPlayer) Previous() error {
+	if lp.player == nil {
+		return errors.New("A list player must be initialized first")
+	}
+
+	// TODO: returns an error ( 0 if nothing is previous)
```

Comment:
I left these TODOs here (also in the Next() method), in part because I'm not sure how to handle the `libvlc` errors that will come out. In any case I found it didn't matter if there was nothing _previous_ and an error returned (didn't matter for my use case, that is), but perhaps we'd want to return this error too.